charts/perses/values.yaml

```diff
@@ -226,3 +226,13 @@ datasources:
   #       kind: PrometheusDatasource
   #       spec:
   #         directUrl: https://prometheus.demo.do.prometheus.io
+
+
+testFramework:
+  enabled: true
```

Comment:
Shouldn't it be false by default ? 
Somebody reusing the chart will by default don't want to create test pods

Reply:
I really appreciate your questions:) 
Let me clarify how the [helm chart test](https://helm.sh/docs/topics/chart_tests/) work. 
The test resources are handled differently from regular chart resources. Even with testFramework.enabled set to true, the test pods won't be created during regular helm install or helm upgrade operations. The test resources will be only bundled inside.  They only come into play when you explicitly run helm test <release_name>.

When tests are executed:
1. Test resources are created temporarily
2. After successful test completion, these resources are automatically cleaned up
3. If a test fails, the test pod is intentionally preserved to allow for debugging

Reply:
Oh great! Thanks for taking time to answer all my questions. 
Then if nothing is created on helm install, I'm fine 🙂